assets/js/main.js

```diff
@@ -222,4 +222,40 @@
   window.addEventListener('load', navmenuScrollspy);
   document.addEventListener('scroll', navmenuScrollspy);
 
-})();
\ No newline at end of file
+})();
+
+function toggleChatbot() {
+  const chatbot = document.getElementById('chatbot');
+  chatbot.style.display = chatbot.style.display === 'flex' ? 'none' : 'flex';
+}
+
+function closechat() {
+  const closec = document.getElementById('chatbot');
+  closec.style.display =  closec.style.display === 'flex' ? 'none': 'flex';
+}
+
+// Send message to chatbot
+function sendMessage() {
+  const userInput = document.getElementById('user-input').value;
+  if (userInput.trim()) {
+    // Display user message
+    const userMessageDiv = document.createElement('div');
+    userMessageDiv.classList.add('message', 'user-message');
+    userMessageDiv.textContent = userInput;
+    document.getElementById('chatbot-body').appendChild(userMessageDiv);
+
+    // Display bot response
+    setTimeout(() => {
+      const botMessageDiv = document.createElement('div');
+      botMessageDiv.classList.add('message', 'bot-message');
+      botMessageDiv.textContent = "Hi! I'm Bob, Mauren's assistant. I'm always online and ready to help.";
+      document.getElementById('chatbot-body').appendChild(botMessageDiv);
```

Comment:
I am not sure about the this method of creating a dialog flow, I suggest that you use If-else statements which would make it easier for you to enhance the dialog between you and the chatbot